src/Leaves/Table.php

```diff
@@ -190,7 +190,7 @@ public function exportList()
             }
 
             foreach ($columns as $column) {
-                $data[$column->label] = $column->getCellContent($item, $decorator);
+                $data[$column->label] = htmlspecialchars_decode($column->getCellContent($item, $decorator));
```

Comment:
Interesting - although I wonder is there an (easy) way to get the cell content without the encoding in the first place? Perhaps getCellContent could call another method  which this function could call directly to get the content without encoding? Haven't looked at the code so it might not turn out to be that simple

Reply:
This has been changed based on our offline chat. PR updated.